components/raftstore/src/store/peer.rs

```diff
@@ -4976,16 +5029,15 @@ where
             Some(ForceLeaderState::ForceLeader { .. })
         )
     }
-
     pub fn unsafe_recovery_maybe_finish_wait_apply(&mut self, force: bool) {
         if let Some(UnsafeRecoveryState::WaitApply { target_index, .. }) =
             &self.unsafe_recovery_state
         {
             if self.raft_group.raft.raft_log.applied >= *target_index || force {
                 if self.is_force_leader() {
                     info!(
-                        "Unsafe recovery, finish wait apply";
-                        "region_id" => self.region().get_id(),
+                       "Unsafe recovery, finish wait apply";
```

Comment:
need run `make format`

Reply:
it looks format can not resolve it. I have manually removed the prefix space.